src/modules/AuthenticationModule/components/Login/Login.tsx

```diff
@@ -1,16 +1,84 @@
-import logo from "../../../../assets/PMS 3.svg";
+import React, { useContext } from 'react'
+import logo from '../../../../assets/PMS 3.png'
+import { Link, useNavigate } from 'react-router-dom'
+import { useForm } from 'react-hook-form';
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import { AuthContext } from '../../../SharedModule/components/AuthContext/AuthContext';
 
+export default function Login() {
 
+  const {baseUrl} = useContext(AuthContext)
 
+  const { register, handleSubmit, formState: { errors } } = useForm()
 
-const Login = () => {
+  const navigate = useNavigate()
 
+  const onSubmit = async (data:any) => {
+    try {
+      const response = await axios.post(`${baseUrl}/Users/Login`, data)
+      localStorage.setItem('token' , response.data.token)
+      console.log(response);
+         toast.success("logged in successfully", {
```

Comment:
We need to provide the message from the backend and the default message as a fallback if there was no message coming from the backend. The same goes for the error.